test/services/GovSpendServiceTest.scala

```diff
@@ -125,6 +125,25 @@ class GovSpendServiceTest extends UnitSpec with MockitoSugar with ScalaFutures {
       result("UkContributionToEuBudget") shouldBe 0.70
     }
 
+    "return correct amounts for 2019" in {
+      val result: Map[String, BigDecimal] = GovSpendService.govSpending(2019)
+
+      result("Welfare") shouldBe 23.50
```

Comment:
This would throw an exception if the value does not exist in the map. You would get a more meaningful output if you used get and compared to a `Some`
```suggestion
      result.get("Welfare") shouldBe Some(23.50)
```

Reply:
done